handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,59 @@
+
+1. We have confidence that our centralized logging infrastructure is production-ready, and we have relevant tooling and testing environments configured.
+2. We're confident that we're ingesting logs from all services that are tightly coupled to our Cloud deployment. This is anything that directly interacts with our cloud deployment with elevated privileges, or directly allows authentication to these services.
+3. Additional continuous scanning and alerting is implemented.
+   1. Continuously scan our sourcecode for vulnerabilities (SAST), and document triage and patching processes.
+   2. Continuously monitor our 3rd party dependencies for vulnerabilities and outdated versions.
+4. We're ingesting logs from all services loosely coupled to our Cloud deployment. This is anything that cannot independently impact Cloud, but may be leveraged to gain access, extract sensitive information, or otherwise negatively impact cloud.
+5. We have confidence that the logs we're ingesting provide enough useful information about their context. This is a prerequisite for normalizing our log format.
+6. Normalize log format in our centralized log storage so that it is easier to correlate and search. This is a prerequisite for creating automated alerts from the logs.
+7. Create alerts and dashboards to automate the process of investigating events of interest (e.g., detect and alert on a spike of failed login attempts to a single account, or across Sourcegraph Cloud as a whole).
+8. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). We then advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to this [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+9. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+10. Document and publish our security practices so that our customers can review and audit them.
+11. Achieve a [SOC 2 type 1](#soc-audit-readiness) certification.
+12. Validate that we are GDPR compliant.
```

Comment:
Similar for GDPR but this really depends on the timeline. It's hard to commit to this without a DPO, etc.

Reply:
Agreed RE: conversation in team sync. Removed.